features/include/pcl/features/impl/multiscale_feature_persistence.hpp

```diff
@@ -91,10 +91,10 @@ pcl::MultiscaleFeaturePersistence<PointSource, PointFeature>::computeFeaturesAtA
   features_at_scale_.reserve (scale_values_.size ());
   features_at_scale_vectorized_.clear ();
   features_at_scale_vectorized_.reserve (scale_values_.size ());
-  for (std::size_t scale_i = 0; scale_i < scale_values_.size (); ++scale_i)
+  for (float & scale_value : scale_values_)
```

Comment:
Is the check for & not applied everywhere? I think it should be without the space, towards float?

Reply:
Applies other places as well.

Reply:
We have it in our clang-format config, but that isn't [enforced everywhere yet](https://github.com/PointCloudLibrary/pcl/blob/master/.dev/format.sh#L11)